repository-data/webfiles/src/main/resources/site/freemarker/nationalindicatorlibrary/indicator.ftl

```diff
@@ -115,19 +115,32 @@
             </#list>   
         </section>     
 
-        <#if indicator.attachments?has_content>
+
+        <#if indicator.hasAttachments()>
             <section id="resources" class="push-double--bottom">
                 <h2><strong><@fmt.message key="headers.resources"/></strong></h2>
-            
-                    <ul data-uipath="nil.indicator.resources">
-                        <#list indicator.attachments as attachment>
-                            <li class="attachment">
-                                <a title="${attachment.text}" href="<@hst.link hippobean=attachment.resource/>" onClick="logGoogleAnalyticsEvent('Download attachment','Indicator','${attachment.resource.filename}');">${attachment.text}</a>
-                                <span class="fileSize">(<@formatFileSize bytesCount=attachment.resource.length/>)</span>
-                            </li>
-                        </#list>
-                    </ul>
-                
+                        <ul data-uipath="nil.indicator.resources">
+                            <#if indicator.details.qualityStatementUrl?has_content>
+                                <li class="attachment">
+                                    <a href="${indicator.details.qualityStatementUrl}" target="_blank"><@fmt.message key="headers.qualityStatement"/></a>
```

Comment:
Do you want GA event to be fired for this?

Reply:
I don't see why not. Will add for consistency.